File: python/7 kyu/30.py
# https://www.codewars.com/kata/5981a139f5471fd1b2000071/train/python

# Passed

def task(w, n, c):
    worker_map = {
        'monday': 'James',
        'tuesday': 'John',
        'wednesday': 'Robert',
        'thursday': 'Michael',
        'friday': 'William'
    }

    return f'It is {w} today, {worker_map[w.lower()]}, you have to work, you must spray {n} trees and you need {n * c} dollars to buy liquid'

File: nodejs/7 kyu/30.js
// https://www.codewars.com/kata/589422431a88082ea600002a/train/javascript

// Passed

function digitDegree(n) {
  let result = 0;
  while (String(n).length > 1) {
    result++;
    n = String(n)
      .split("")
      .reduce((a, b) => Number(a) + Number(b));
  }

  return result;
}

const output = digitDegree(547);
console.log(output);